### PR TITLE
Add website attestation

### LINKED ISSUE #1401: Website attestation
Would be useful to prove you have control over a particular website, perhaps using the same verification technique as Google Web Console, ie add a meta tag to your site, or add a TXT entry to DNS. A link to the website could then be included in the user's profile.

### PR DESCRIPTION
**Pull request by shahthepro**
Closes #1401 

### Checklist:

- [ ] UX Review
- [x] Add icons to DApp

### CODE REVIEW

File: packages/graphql/src/configs/localhost.js [hunk 1]
```diff
@@ -12,7 +12,7 @@ const config = {
   providerWS: `ws://${HOST}:8545`,
   ipfsGateway: `http://${HOST}:8080`,
   ipfsRPC: `http://${HOST}:5002`,
-  bridge: 'https://bridge.dev.originprotocol.com',
```
Comment (nick) [inline packages/graphql/src/configs/localhost.js]: Can we switch this back for now?

Comment (shahthepro) [reply packages/graphql/src/configs/localhost.js]: Will revert it.

**Comment by shahthepro** [comment]
> If I fail to upload the file and press Continue, I don't see an error. Also see this in the console:
> 
> image

Will work on that. Thanks

**Comment by shahthepro** [comment]
@nick Have made those changes.

**Comment by micahalcorn** [comment]
Please let me take a look at this before we merge 😁

File: infra/bridge/src/controllers/website.js [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+      pubAuditableUrl: {
+        proofUrl: remoteFileURL
+      }
+    },
+    domain: {
+      verified: true
+    }
+  }
+
+  const attestation = await generateAttestation(
```
Comment (nick) [inline infra/bridge/src/controllers/website.js]: Can we wrap this in a try/catch. I had not run the migration which caused an unhandled promise rejection, causing the request to not return a response:

```
(node:47398) UnhandledPromiseRejectionWarning: SequelizeDatabaseError: invalid input value for enum enum_attestation_method: "WEBSITE"
    at Query.formatError (/Users/nick/Projects/origin/node_modules/sequelize/lib/dialects/postgres/query.js:354:16)
    at query.catch.err (/Users/nick/Projects/origin/node_modules/sequelize/lib/dialects/postgres/query.js:71:18)
```

**Comment by nick** [comment]
Here's what the UI looks like:

Screen Shot 2019-05-09 at 12 16 41 PM
Screen Shot 2019-05-09 at 12 16 48 PM
Screen Shot 2019-05-09 at 12 17 11 PM
Screen Shot 2019-05-09 at 12 17 25 PM
Screen Shot 2019-05-09 at 12 17 56 PM
Screen Shot 2019-05-09 at 12 18 02 PM

**Comment by nick** [comment]
@shahthepro icons are [here](https://app.zeplin.io/project/59fa2311bac7acbc8d953da9/screen/5cd48a0536dbe434b44b0c90)

**Comment by nick** [comment]
Can we also split up 'Download' and 'Continue' into two separate steps? So when the user clicks the 'Download' button it then shows the 'Continue' step?

**Comment by shahthepro** [comment]
@nick 
> Can we also split up 'Download' and 'Continue' into two separate steps? So when the user clicks the 'Download' button it then shows the 'Continue' step?

Done.

> @shahthepro icons are [here](https://app.zeplin.io/project/59fa2311bac7acbc8d953da9/screen/5cd48a0536dbe434b44b0c90)

A verified icon with green background is missing. Have posted a comment on Zeplin.

@micahalcorn 

> Can this be wired off or something until we have some time to figure out the UI/UX? 

I guess this attestation is not a priority now. Even if we merge it, it will be hidden, in the DApp, until we set the `ENABLE_WEBSITE_ATTESTATION` env to "true". If @nick has no objections, we can hold this off.

**Pull request merged**